Dispatcher/Networking/FirebaseManager.swift

```diff
@@ -30,4 +30,24 @@ class FirebaseManager {
             }
         }
     }
+    
+    func logoutUserFromFirebase(completionHandler: @escaping (_ errorMsg: String?) -> Void) {
+        do {
+          try Auth.auth().signOut()
+            completionHandler(nil)
+        } catch let signOutError as NSError {
+            completionHandler(signOutError.localizedDescription)
+        }
+    }
+    
+    func getLastLoginOfCurrentUser() throws -> String {
+        let previousLogin = UserDefaultsManager().fetchDataFromUserDefaults(wantedItemKey: UserDefaultsKeys.lastLoginOfUserDefualtsKey)
```

Comment:
Why not use the last login property from Firebase?
(Also - if you use user defaults then this function shouldn't be inside the FirebaseManager)